.github/workflows/site.yml

```
@@ -0,0 +1,69 @@
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pages: read
+    steps:
+      - uses: actions/checkout@v7
+        with:
+          persist-credentials: false
+      - id: nodeversion
+        run: echo "version=$(grep nodejs .tool-versions | sed -e 's/[^[:space:]]*[[:space:]]*//')" >> $GITHUB_OUTPUT
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:90: Double quote to prevent globbing and word splitting

(shellcheck)